### ISSUE TITLE
linked media sources of TVs

### ISSUE THREAD

**Issue by degoya** [modmore/Gitify#43]
How to extract the Media Sources from TVs?

i've got the follow in my .gitify file so far

  template_variables:
    class: modTemplateVar
    primary: name
  template_variables_access:
    class: modTemplateVarTemplate
    primary: [tmplvarid, templateid]
  mediasources:
    class: modMediaSource
    primary: name

can't find the linked mediasources

**Comment by Mark-H** [comment]
It looks like the modTemplateVar.source field should contain the ID of the media source?

**Comment by Mark-H** [comment]
For the TV media source relations, you'll need this in your .gitify's data.. It requires 0.5.2 (more specifically, the fix of #65), but seems to do the trick:

```` 
  mediasource_elements:
    class: sources.modMediaSourceElement
    primary: [source, object_class, object, context_key]
````
